### PR TITLE
Release notes for 5.6.9

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_5.6.md [hunk 1]
```diff
@@ -28,6 +28,63 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
 
+## Release 5.6.9-1
+
+**Release Date:** July 13, 2023
+
+### Resolved Issues
+
+- **I95-47960 Incorrect progress message for `show dns resolutions`:** The progress message for this command now correctly displays `Retrieving dns resolutions...`.
+------
+- **I95-48931 Service area Highway crash:** Now prevent crashing in SSR's highway process in unusual race conditions when a session's flow is removed before the session is fully established.
```
[the remark this answers is not in the document]
Comment (hadrielk) [reply docs/release_notes_128t_5.6.md:39]: or even "rare"?

File: docs/release_notes_128t_5.6.md [hunk 2]
```diff
@@ -28,6 +28,63 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-49587 ICMP session classification improvement:** The application lookup for ICMP sessions has been optimized to accurately identify the correct service.
```
Comment (hadrielk) [inline docs/release_notes_128t_5.6.md:41]: Lol, that's not _exactly_ "optimized".

How about "fixed".

File: docs/release_notes_128t_5.6.md [hunk 3]
```diff
@@ -28,6 +28,63 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-51296 Show Time in Status in the show assets detail view:** The asset Time in Status field has been added to the Detail view.
+------
+- **I95-51359 Unable to set the OSPF MTU:** Added the ability for users to set the MTU to a non-default value.
+------
+- **I95-51403 GUI displays download in progress even after the download is complete:** Resolved an issue where a download success event is never created even though the version shows as downloaded in the software versions.
+------
+- **I95-51427 GUI not displaying all the version information:** The GUI About page now displays additional version information previously only displayed in the PCLI `show system version detail.`
+------
+- **I95-51621 Provide a way for the user to display the history of configuration stored on disk:** Users can now use the `show diskcache` command to retrieve metadata information about the current running configuration as well as the cached configurations that are still present on the system.
```
Comment (hadrielk) [inline docs/release_notes_128t_5.6.md:63]: They named such a command "`show diskcache`"?!?

Ugh.

File: docs/release_notes_128t_5.6.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.6.md:39]: I like rare.

File: docs/release_notes_128t_5.6.md (hunk 3 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.6.md:63]: checking...

File: docs/release_notes_128t_5.6.md [hunk 4]
```diff
@@ -60,8 +60,6 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
 ------
 - **I95-51427 GUI not displaying all the version information:** The GUI About page now displays additional version information previously only displayed in the PCLI `show system version detail.`
 ------
-- **I95-51621 Provide a way for the user to display the history of configuration stored on disk:** Users can now use the `show diskcache` command to retrieve metadata information about the current running configuration as well as the cached configurations that are still present on the system.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.6.md]: Was this supposed to be removed?

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by hadrielk — APPROVED** [review]

**Pull request merged**